glustercli/cmd/root.go

```diff
@@ -96,6 +97,19 @@ var (
 	flagTimeout    uint
 )
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print glusterd and gluster-cli version",
+	Run: func(cmd *cobra.Command, args []string) {
+		gd, err := client.Version()
+		if err == nil {
+			fmt.Println("glusterd version:", gd.GlusterdVersion)
+			fmt.Println("glusterd api-version:", gd.APIVersion)
+		}
```

Comment:
I'd like to see an else condition here that prints something like "unable to fetch server version" or something along those lines. If the command fetches the server version by default I don't think we ought to hide a failure condition if the client does not successfully get the server's version.

That said, it may also make sense to add a -S/--no-server flag for the subcommand so that a user who does not have a server to query (yet) can still get the version of the tool itself.

IOW, don't hide the error condition but provide the user a way to avoid hitting that error condition when they know better.

Reply:
I can add else statement to the print error message only verbose flag is sent and discard printing gluster-cli version. IMO, it's not a good idea to print both error message and an actual response.

adding ```--no-server``` flag makes sense to discard error message while fetching server version fails.

@aravindavk  @prashanthpai, want to know your thoughts on this